logic/src/commonMain/kotlin/com/wire/kalium/logic/data/notification/LocalNotification.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.notification
+
+import com.wire.kalium.logic.data.id.ConversationId
+
+data class LocalNotificationConversation(
```

Comment:
Can we add some documentation about what local means? What I understand is that are Kalium self generated notifications, independent from platform push up notifications?